### PR TITLE
Update readme with changes from UI changes and updated screenshots

### PR DESCRIPTION
**Pull request by pbking**
Updates the readme files and screenshots to include changes from UI refactor.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -42,43 +44,59 @@ Make changes to your site styles, fonts and templates using the Editor.
 = Step 3 – Export =
 Still in the WordPress Editor, navigate to the Create Block Theme menu at the top of the Editor.
 
-To simply save recent changes made in the Editor to the currently active theme or export the theme:
+To save recent changes made in the Editor to the currently active theme
```
Comment (mikachan) [inline readme.txt]:
```suggestion
To save recent changes made in the Editor to the currently active theme:
```

File: readme.txt [hunk 2]
```diff
@@ -42,43 +44,59 @@ Make changes to your site styles, fonts and templates using the Editor.
...
+
+To create a new Clone of the current theme or to create a Child of the current theme:ons for the currently active theme:
+
+- Click "Create Theme"
+- Click "Clone Theme" to create a new Theme based on the active theme with your changes
+- Click "Create Child Theme" to create a new Child Theme with the active theme as a parent with your changes
+
+5. Create Theme panel 1 in the WordPress Editor
+
+6. Create Theme panel 2 in the WordPress Editor
```
Comment (mikachan) [inline readme.txt:95]: It looks like this is referencing a new screenshot but I don't see the new screenshot as part of this branch.

File: readme.txt [hunk 3]
```diff
@@ -42,43 +44,59 @@ Make changes to your site styles, fonts and templates using the Editor.
...
 
 Many of these options are also available under the older, deprecated Create Block Theme page under Appearance > Create Block Theme.
 
-5. Page under Appearance > Create Block Theme
+7. Page under Appearance > Create Block Theme
```
Comment (mikachan) [inline readme.txt:99]: Similar here - looks like this is a new screenshot but there's no screenshot-7.jpg file.

File: README.md [hunk 4]
```diff
@@ -79,23 +79,47 @@ Make changes to your site styles, fonts and templates using the Editor.
...
+To create a variation:
+
+- Select "Create Theme Variation"
+- Provide a name for the new Variation
+- Click "Create Theme Variation"
+
+A new variation will be created.
 
-To create a new theme, child theme, or style variation, select "Create Theme" to access the following options for the currently active theme:
+To create a new Clone of the current theme or to create a Child of the current theme:ons for the currently active theme:
```
Comment (mikachan) [inline README.md]:
```suggestion
To create a new Clone of the current theme or to create a Child of the current theme:
```

**Review by mikachan — COMMENTED** [review]
I noticed a few small issues but otherwise this is looking good.

File: readme.txt (hunk 2 above)
Comment (pbking) [reply readme.txt:95]: Oops!  I'll add them.

**Review by mikachan — APPROVED** [review]
Thanks, these changes look good!

**Pull request merged**